src/yadism/coefficient_functions/light/nlo/g1.py

```diff
@@ -0,0 +1,27 @@
+import numba as nb
+import numpy as np
+from eko.constants import TR, CF
+
+from . import f2
```

Comment:
```suggestion

```

Reply:
it's still used for the local and singular part.

Reply:
My bad! Missed it by looking at the commit only.